ansible/roles/pgo-operator/files/pgo-configs/pgbouncer-template.json

```diff
@@ -63,9 +55,6 @@
                     }, {
                         "name": "PG_PRIMARY_SERVICE_NAME",
                         "value": "{{.PrimaryServiceName}}"
-                    }, {
-                        "name": "PG_REPLICA_SERVICE_NAME",
```

Comment:
I see this environment variable referenced in [the docs](https://github.com/CrunchyData/postgres-operator/blob/54229ff17e583274025c120380c699e4de8b2e1e/hugo/content/pgo-client/common-tasks.md#connection-pooling-via-pgbouncer); is it safe to remove?

> If you want [PgBouncer] to have an entry for the replica service, add the following configuration to pgbouncer.ini…

Reply:
The methodology has shifted now that we have the `*` flag being used to match all available databases.

I went ahead and remove the line from the documentation.